sdk/storage/storage-blob/tests.yml

```diff
@@ -6,6 +6,7 @@ extends:
     PackageName: "@azure/storage-blob"
     ResourceServiceDirectory: storage
     TimeoutInMinutes: 90
+    Location: canadacentral
```

Comment:
Should be `ArmTemplateLocation: canadacentral`

Reply:
Got it.